conf/client.conf

```diff
@@ -129,6 +129,9 @@ global.fileIOSplitMaxSizeKB=64
 #
 ################# log相关配置 ###############
 #
+# enable logging or not
+global.logging.enable=True
```

Comment:
Why need this configuration item？

Reply:
because in previous version, curve-client will always init glog in which it will set a different logging dir, then curve-fuse's logging destination will be affected.